eos/utils/expression-evaluator.hh

```diff
@@ -37,6 +38,10 @@ namespace eos::exp
 
             double visit(ObservableExpression & e);
 
+            double visit(ParameterNameExpression &);
```

Comment:
For consistency, a `e` could be added here.

Reply:
Actually, I disagree. ``ExpressionEvaluator::visit(ParameterNameExpression &)`` does not use its argument, hence it should be unnamed to avoid the warning that the argument is not used.

Reply:
You will probably not get a Warning since it is in the header file, but ok :)